gammapy/catalog/tests/test_fermi.py

```diff
@@ -107,6 +107,16 @@ def test_spectral_model(self, ref):
         assert_quantity_allclose(dnde, ref['dnde'])
         assert_quantity_allclose(dnde_err, ref['dnde_err'])
 
+    @pytest.mark.parametrize('ref', SOURCES_3FGL, ids=lambda _: _['name'])
```

Comment:
@cdeil - I have tons of issues with these tests when checking on the deprecation warnings.

Actually I don't see how this ever worked as `SOURCES_3FGL` above has no `'name'` key just `'idx'`, but then if I change `name` to `idx` it all fall over. Are there names for those sources that can be used here? Why are these tests pass for you?